wget http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/linuxthreads/sysdeps/alpha/pt-machine.h.diff?r1=1.9&r2=1.10&cvsroot=glibc

Should fix error

internals.h: In function `pthread_setcancelstate':
internals.h:381: error: asm-specifier for variable `__self' conflicts with asm clobber list
make[2]: *** [/home/dank/crosstool-0.28/build/alpha-unknown-linux-gnu/gcc-3.4.1-glibc-2.2.5/build-glibc/linuxthreads/cancel.o] Error 1
make[2]: Leaving directory `/home/dank/crosstool-0.28/build/alpha-unknown-linux-gnu/gcc-3.4.1-glibc-2.2.5/glibc-2.2.5/linuxthreads'

===================================================================
RCS file: /cvs/glibc/libc/linuxthreads/sysdeps/alpha/pt-machine.h,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- libc/linuxthreads/sysdeps/alpha/pt-machine.h	2002/04/06 04:05:13	1.9
+++ libc/linuxthreads/sysdeps/alpha/pt-machine.h	2002/05/20 20:09:34	1.10
@@ -76,7 +76,7 @@
 #define THREAD_SELF \
 ({									      \
   register pthread_descr __self __asm__("$0");				      \
-  __asm__ ("call_pal %1" : "=r"(__self) : "i"(PAL_rduniq) : "$0");	      \
+  __asm__ ("call_pal %1" : "=r"(__self) : "i"(PAL_rduniq));		      \
   __self;								      \
 })
 
